packages/apps/desktop-viewer-test/src/frontend/components/routes/ViewerRoute.tsx

```diff
@@ -69,7 +68,7 @@ export const ViewerRoute = () => {
                   getLabel: () => ModelsTreeComponent.getLabel(),
                   render: (props) => (
                     <ModelsTreeComponent
-                      getSchemaContext={getSchemaContext}
+                      getSchemaContext={(iModel) => iModel.schemaContext}
```

Comment:
Adding @saskliutas to confirm, but I think we're expecting this function to be memoized using `React.useCallback`, otherwise it will cause excessive re-renders.

Reply:
It should be ok at this level as `render` is called only when tree is changed from selector.